src/main/java/dev/sigstore/rekor/client/RekorEntry.java

```diff
@@ -33,6 +35,36 @@ public interface RekorEntry {
   interface Verification {
     /** Return the signed entry timestamp. */
     String getSignedEntryTimestamp();
+
+    @Nullable
+    InclusionProof getInclusionProof();
```

Comment:
Could we just use optional instead of nullable though?

Reply:
doh. Still learning the habits. fixed.